include/igl/find_cross_field_singularities.cpp

```diff
@@ -38,7 +37,7 @@ IGL_INLINE void igl::find_cross_field_singularities(const Eigen::PlainObjectBase
     if (V_border[vid])
       continue;
 
-    int missmatch=0;
+    int mismatch=0;
```

Comment:
Can we put spaces around operators: i.e. `mismatch = 0`

Reply:
A lot of existing code doesn't put extra spaces between operators. I agree with you that I prefer pep8-style spacing for expression, but we don't have a specific guideline for it and this is too much to enforce (just mimic existing code we have).

Reply:
Again, @copyme, feel free to not add spaces since it's not in our style guidelines. That said, I do think it's cleaner and you can add them if you want.